discovery-provider/src/queries/get_attestation.py

```diff
@@ -176,3 +178,29 @@ def get_attestation(
         attestation_bytes, shared_config["delegate"]["private_key"]
     )
     return (shared_config["delegate"]["owner_wallet"], signed_attestation)
+
+
+ADD_SENDER_MESSAGE_PREFIX = "add"
```

Comment:
will handle updates to these in a subsequent PR. thanks for including this work now